app/javascript/components/App.jsx

```diff
@@ -6,7 +6,9 @@ import Recipes from "./example/ExampleRecipes";
 import Recipe from "./example/ExampleRecipe";
 import NewRecipe from "./example/ExampleNewRecipe";
 import Header from "./Header";
-import EventsPage from "./pages/events/EventsPage";
+import EventsPage from "./pages/EventsPage";
+import MembersPage from "./pages/MembersPage";
+import Footer from './Footer';
```

Comment:
Hey you're deleting other's code here, please be more careful in the future
```suggestion
import EventsPage from "./pages/events/EventsPage";
import Footer from './Footer';
```